app/main.py

```diff
@@ -1,7 +1,17 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
 
+import requests
 
-if __name__ == "__main__":
-    get_weather()
+API_KEY = os.environ.get("WEATHER_API_KEY")
+BASE_URL = "http://api.weatherapi.com/v1/"
+CITY = "Paris"
+
+
+def get_weather() -> str:
+    payload = {"key": API_KEY, "q": CITY}
+    get_weather_url = BASE_URL + "current.json"
+    data = requests.get(get_weather_url, params=payload).json()
+    return (f"{data['location']['name']}/{data['location']['country']} "
+            f"{data['location']['localtime']} "
+            f"Weather: {str(data['current']['temp_c'])} Celsius, "
+            f"{data['current']['condition']['text']}")
```

Comment:
what if API returns 400 401 500 or other status code that returns errors, not data as you expect here? Let's handle this case as well

Reply:
Also this function should print the result, not return